packages/calcite-components/src/components/notice/notice.tsx

```diff
@@ -75,6 +73,12 @@ export class Notice
   /** When `true`, the component is visible. */
   @Prop({ reflect: true, mutable: true }) open = false;
 
+  @Watch("open")
+  openHandler(value: boolean): void {
+    onToggleOpenCloseComponent(this);
+    value ? this.open : !this.open;
```

Comment:
what is value doing here? Its not updating anywhere.

Reply:
`openHandler(this.open) is called in the `connectedCallback` for all of these.

Reply:
The line of code isn't doing anything